contracts/governance/Governor.sol

```diff
@@ -47,6 +47,7 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
     }
     // solhint-enable var-name-mixedcase
 
+    bytes32 private constant _ALL_PROPOSAL_STATES_BITMAP = bytes32(2 ** uint256(type(ProposalState).max) - 1);
```

Comment:
```suggestion
    bytes32 private constant _ALL_PROPOSAL_STATES_BITMAP = bytes32(2 ** uint8(type(ProposalState).max) - 1);
```

Reply:
Not sure that this is right. I'm concerned that `2 ** uint8(x)` will be typed as `uint8` and potentially overflow?

Is there a benefit to this change?

Reply:
IMO its more explicit about the exponent being <= 255 and so the exponentiation part not overflowing.
Also, I checked and `2**uint8(x)` is not typed as uint8.

Reply:
lets forget that, its not important